📖 [bento][amp-accordion] Update bento documentation to match template

krdwan opened the pull request:
Update bento docs to match template.

Referenced template PR: https://github.com/ampproject/amphtml/pull/31944

caroqliu commented on extensions/amp-accordion/amp-accordion.md:
```diff
@@ -222,100 +215,167 @@ use its corresponding `id` as the value.
+Fully valid AMP pages use the AMP layout system to infer sizing of elements to create a page structure before downloading any remote resources. However, Bento use imports components into less controlled environments and AMP's layout system is inaccessible.
 
-[/filter] <!-- formats="websites" -->
+## Version notes
```
Should this go to the bottom of the file?

krdwan replied:
Thanks, moved to the bottom of the file!

CrystalOnScript commented on extensions/amp-accordion/amp-accordion.md:
```diff
@@ -222,100 +215,167 @@ use its corresponding `id` as the value.
 
-### collapse
-
+**collapse**
```
```suggestion
**collapse()**
```

krdwan replied:
Thanks, updated!

CrystalOnScript commented on extensions/amp-accordion/amp-accordion.md:
```diff
@@ -222,100 +215,167 @@ use its corresponding `id` as the value.
-
-### expand
-
+**expand**
```
```suggestion
**expand()**
```

krdwan replied:
Thanks, updated!

CrystalOnScript commented on extensions/amp-accordion/amp-accordion.md:
````diff
@@ -95,118 +95,111 @@ The example below contains an `amp-accordion` with three sections. The
+</script>
+```
 
+**toggle**
````
```suggestion
**toggle()**
```

krdwan replied:
Thanks, updated!

caroqliu commented on extensions/amp-accordion/amp-accordion.md:
```diff
@@ -222,100 +215,167 @@ use its corresponding `id` as the value.
-
-### expand
-
+**expand**
```
```suggestion
**expand()**
```

krdwan replied:
Thanks, updated!

caroqliu commented on extensions/amp-accordion/amp-accordion.md:
```diff
@@ -222,100 +215,167 @@ use its corresponding `id` as the value.
 
-### collapse
-
+**collapse**
```
```suggestion
**collapse()**
```

krdwan replied:
Thanks, updated!

CrystalOnScript reviewed (changes requested):
This looks nearly perfect! Just a few nits and one re-order request:

Could you move the actions above the events and add a header above each?
```
##### Actions

actions....

##### Events

events...
```

Similar to amp-lightbox.md in this pr
https://github.com/ampproject/amphtml/commit/c1d09c457911f5906debeb04a5f0c14bf8734618

caroqliu commented on extensions/amp-accordion/amp-accordion.md:
````diff
@@ -95,118 +95,111 @@ The example below contains an `amp-accordion` with three sections. The
+</script>
+```
 
+**toggle**
````
```suggestion
**toggle()**
```

krdwan replied:
Thanks, updated!

krdwan commented:
> This looks nearly perfect! Just a few nits and one re-order request:
> 
> Could you move the actions above the events and add a header above each?
> 
> ```
> ##### Actions
> 
> actions....
> 
> ##### Events
> 
> events...
> ```
> 
> Similar to amp-lightbox.md in this pr
> [c1d09c4](https://github.com/ampproject/amphtml/commit/c1d09c457911f5906debeb04a5f0c14bf8734618)

Thanks for the review, I've re-arranged per your comments and addressed nits!!

krdwan commented on extensions/amp-accordion/amp-accordion.md:
````diff
@@ -95,118 +95,111 @@ The example below contains an `amp-accordion` with three sections. The
 ```html
-<amp-accordion animate>
-  <section>
+<head>
````
Thanks, updated!
